packages/native-stack/src/index.tsx

```diff
@@ -8,6 +8,12 @@ export { createNativeStackNavigator } from './navigators/createNativeStackNaviga
  */
 export { NativeStackView } from './views/NativeStackView';
 
+/**
+ * Hooks
+ */
+export { AnimatedHeaderHeightContext } from './utils/AnimatedHeaderHeightContext';
```

Comment:
Well, that's the context, not a hook

Reply:
You're right 😄 removed this export in https://github.com/react-navigation/react-navigation/pull/11663/commits/c1aa5fef1ef2d7cf15898baf611367da094f98a2.